sql-statements/sql-statement-alter-resource-group.md

```diff
@@ -12,22 +12,22 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-alter-resource-group/','/do
 
 ```ebnf+diagram
 AlterResourceGroupStmt:
-	"ALTER" "RESOURCE" "GROUP" IfNotExists ResourceGroupName ResourceGroupOptionList BurstableOption
+   "ALTER" "RESOURCE" "GROUP" IfNotExists ResourceGroupName ResourceGroupOptionList BurstableOption
```

Comment:
Why change the format of these files

Reply:
CI jekins format check requires spaces instead of tabs.